plugins/modules/terraform.py

```diff
@@ -598,7 +600,7 @@ def process_complex_args(vars):
                     ret_out.append('{0}={{{1}}}'.format(k, process_complex_args(v)))
                 elif isinstance(v, list):
                     ret_out.append("{0}={1}".format(k, process_complex_args(v)))
-                elif isinstance(v, (int, float, str, bool)):
+                elif isinstance(v, (int, float, str, bool, type(None))):
```

Comment:
Here I don't have a preference over your solution and
```suggestion
                elif isinstance(v, (int, float, str, bool)) or v is None:
```
Whatever you prefer (in case nobody else cares :) ).